Sources/Build/BuildPlan.swift

```diff
@@ -1919,14 +1919,7 @@ public class BuildPlan {
         // Add search paths from the system library targets.
         for target in graph.reachableTargets {
             if let systemLib = target.underlyingTarget as? SystemLibraryTarget {
-                for flag in self.pkgConfig(for: systemLib).cFlags {
```

Comment:
Are these changes unnecessary now?

Reply:
Yeah, as of https://github.com/apple/swift/pull/37206 this is unnecessary because the API digester and compiler are using the same option definitions